app/mailers/user_mailer.rb

```diff
@@ -71,7 +71,7 @@ def new_phenotype(phenotype,user)
 
   def newsletter(user)
     @user = user
-    mail(:subject => "openSNP.org: participate in a survey on the experiences of sharing genetic data", :to => @user.email)
+    mail(:subject => "openSNP.org: We are now crowdfunding, did server updates, and are interested in your feedback!", :to => @user.email)
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.